server/src/index.ts

```diff
@@ -132,8 +133,33 @@ io.on('connection', async (socket: any) => {
       console.error("[WS] Error calling updateLocation:", error.message)
     }
   })
+  socket.on('updateDisplayName', async (newDisplayName, ack) => {
```

Comment:
These should all be REST API calls. We do not want to make our socket connection stuffed with traffic!

We will handle errors with updating usernames when we get there. For now the general idea is that they will change their name and if it is accepted then it will give them a checkmark or something!

Reply:
Done